packages/unix-errno/unix-errno.0.6.2/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Unix errno types, maps, and support"
+description: """\
+unix-errno can be used with or without ctypes and OCaml's Unix
+module. Without ctypes and Unix, the basic types and functions are
+provided as well as Errno_host containing errno maps for popular
+operating systems. The errno-srcgen tool for generating OCaml source
+representing Errno.Host.t values will also be built. With ctypes and
+Unix, you'll also receive the errno-map tool for outputting the current
+host's errno map and the Errno_unix module containing an errno global
+variable checking function and Unix.error type converters."""
+maintainer: "Xapi Project"
+authors: "David Sheets <sheets@alum.mit.edu>"
+license: "ISC"
+tags: ["errno" "errno.h" "errors" "unix" "syscall"]
+homepage: "https://github.com/xapi-project/ocaml-unix-errno"
+bug-reports: "https://github.com/xapi-project/ocaml-unix-errno/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.8"}
+  "alcotest" {with-test}
+  "result"
+  "ctypes" {>= "0.7.0"}
```

Comment:
```suggestion
  "ctypes" {>= "0.12.0"}
```
Sice you're using `integers` directly in `unix-errno` you need to unsure the `Signed` module from `ctypes < 0.12.0` and the one from integers doesn't clash
```
#=== ERROR while compiling unix-errno.0.6.2 ===================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.14.1 | pinned(https://github.com/xapi-project/ocaml-unix-errno/releases/download/0.6.2/unix-errno-0.6.2.tbz)
# path                 ~/.opam/4.14/.opam-switch/build/unix-errno.0.6.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p unix-errno -j 39
# exit-code            1
# env-file             ~/.opam/log/unix-errno-7-ab69b3.env
# output-file          ~/.opam/log/unix-errno-7-ab69b3.out
### output ###
#       ocamlc unix/.unix_errno.objs/byte/errno_unix.{cmi,cmti} (exit 2)
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I unix/.unix_errno.objs/byte -I /home/opam/.opam/4.14/lib/ctypes -I /home/opam/.opam/4.14/lib/integers -I lib/.errno.objs/byte -I lib_gen/.errno_bindings.objs/byte -I lib_gen/.errno_types.objs/byte -I lib_gen/.errno_types_detected.objs/byte -no-alias-deps -o unix/.unix_errno.objs/byte/errno_unix.cmi -c -intf unix/errno_unix.mli)
# File "unix/errno_unix.mli", line 1:
# Error: The files lib/.errno.objs/byte/errno.cmi
#        and /home/opam/.opam/4.14/lib/ctypes/signed.cmi
#        make inconsistent assumptions over interface Signed
```